packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.3
```

Comment:
There should not be a changelog entry in this package; nothing relevant to publishing has changed.

Reply:
> this should be a bump

Reply:
> How would the ios dependency change if there's no version change here?

I'm not sure what you are asking here; the iOS implementation is a completely separate package.

> Or you mean version bump and no changelog, isn't that kinda weird?

No, a package whose only change is a minor adjustment to an unpublished example file should have no version change or CHANGELOG entry.

Reply:
>I'm not sure what you are asking here; the iOS implementation is a completely separate package.

What I meant was that `google_maps_flutter_ios` will have a new version right, so how would users get this new version, since the dependency on it is through `google_maps_flutter`

Reply:
The same way they would get updates to any transitive dependencies; they would run `flutter pub upgrade`.